addons/expanded-backpack/addon.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Multiple rows in backpack",
+  "description": "Display the contents of the editor backpack in an expandable two-way grid similar to the costume library and increase the size of thumbnails to improve ease of use.",
+  "credits": [
+    {
+      "name": "pumpkinhasapatch",
+      "link": "https://github.com/pumpkinhasapatch"
+    }
+  ],
+  "versionAdded": "1.29.0",
+  "tags": ["editor"],
+  "enabledByDefault": false,
+  "settings": [
+    {
+      "name": "Number of displayed rows",
+      "id": "rows",
+      "type": "integer",
+      "min": 1,
+      "max": 4,
+      "default": 2
+    },
+    {
+      "name": "Improved costume and script previews",
+      "id": "upscale",
+      "type": "boolean",
+      "default": true
+    }
+  ],
+  "customCssVariables": [
+    {
+      "name": "imgsize",
+      "value": {
+        "type": "ternary",
+        "source": {
+          "type": "settingValue",
+          "settingId": "upscale"
+        },
+        "true": "95%",
+        "false": "32px"
+      }
+    },
+    {
+      "name": "bgcolor",
+      "value": {
+        "type": "ternary",
+        "source": {
+          "type": "settingValue",
+          "settingId": "upscale"
+        },
+        "true": "white",
+        "false": "none"
```

Comment:
`none` is a valid value for `background` but not for `background-color`. I think this should be `transparent`.

Reply:
Thanks, leaving this open to discuss the buggy resizing issue above.

I've also noticed that the vertical gap between rows changes depending on the height of the backpack element.
This will definitely need to be fixed before merging, and is especially noticeable on large screens or when using rows=4.
Not sure what the exact cause of it is though and might need to change how resizing the height works.

https://user-images.githubusercontent.com/113677241/205540655-621fc40d-f1f4-4bcf-bc64-00138bbd0c56.mp4

Reply:
I didn't have the latest changes. The spacing issue is caused by this selector: `[class^="backpack_backpack-item"]`

It should be `[class*="backpack_backpack-item"]` because that class doesn't appear at the beginning of the attribute value.

Reply:
> that class doesn't appear at the beginning of the attribute value

My bad, sorry!

Reply:
Is this fixed?

Reply:
> This can be fixed by adding a userscript that triggers a `resize` event on the `window` when the addon is enabled or disabled or a setting is changed.

did some digging through the addon api docs and the hide-stage addon userscript (which seems to have a similar issue) and i think i've found a solution